src/components/Profile/ProfileLikes.tsx

```diff
@@ -19,7 +20,11 @@ const ProfileLikes = () => {
           },
         }
       );
-      dispatch(setPosts(result.data as Post[]));
+      const modified = result.data.map((entry: any) => ({
+        ...entry,
+        timestamp: formatTimestamp(entry),
+      }));
+      dispatch(setPosts(modified as Post[]));
     };
     fetchPosts();
   }, [dispatch, user]);
```

Comment:
Because `formatTimestamp` exists outside the scope of this `useEffect` we should add it to the dependency array
```suggestion
  }, [dispatch, user, formatTimestamp]);
```